models/br_inpe_queimadas/code/microdados/extraction.py

```
@@ -1,63 +1,71 @@
 import pandas as pd
 import requests
 from bs4 import BeautifulSoup
 
+# URL base dos dados mensais no servidor do INPE
+MONTH_URL = "https://dataserver-coids.inpe.br/queimadas/queimadas/focos/csv/mensal/Brasil/"
+
 
-def get_html_table(url, n_rows):
+# Faz scraping da tabela HTML do servidor INPE e retorna os arquivos disponíveis como DataFrame
+def get_html_table(url, n_rows=None):
     response = requests.get(url)
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)